packages/documentation/copy/en/developer/Getting_Started_Linux.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Getting started on Linux 
+layout: docs
+permalink: /docs/handbook/getting-started-linux
+oneline: "Quick start with Lingua Franca on Linux"
+preamble: >
+---
+
+# Getting started on Linux
+
+## Installation
+
+### Debian & Other
+
+Sadly there are currently no packaged version of the lingua-franca compiler in debian distributions so there is
+really only the manual way. But we offer prebuild binaries [here](https://github.com/lf-lang/lingua-franca/releases). 
+Please note that you need a modern version of java 17 or greater.
+
+If you want to compile the lingua-franca compiler from scratch there is a [guide](/docs/handbook/download) for that. 
+Alternatively you can use the prebuild lfc.
+
+**Short instructions for downloading lfc**
+```bash
+    wget https://github.com/lf-lang/lingua-franca/releases/download/v0.2.0/lfc_0.2.0.tar.gz
+    tar xf lfc_0.2.0.tar.gz
+    export PATH="$(pwd)/lfc_0.2.0/bin/":$PATH // maybe add that to you bashrc/zshrc aliases
+    lfc ./src/yourproject.lf
+```
+
+### Arch
+
+There are packaged version of [lfc](https://aur.archlinux.org/packages/lfc-bin) and [epoch](https://aur.archlinux.org/packages/epoch-bin) the arch user repository.
+
+```bash
+    git clone https://aur.archlinux.org/packages/lfc-bin
```

Comment:
Is there a reason for manually downloading and building the package instead of using an AUR helper like `yay`?

Reply:
There shouldn't be :-) We should write some distro-specific instructions for cases where installation can be done with a simple one-liner.